setup.py

```diff
@@ -17,7 +17,7 @@ def read(fname):
 
 setup(
     name="shoobx.mocks3",
-    version='1.4.3.dev0',
+    version='1.5.0.dev0',
```

Comment:
BTW, if you use zest.releaser it will set this version automatically for you. :-)